lib/ValidationError.ts

```diff
@@ -55,36 +66,79 @@ function fromZodIssue(
   return issue.message;
 }
 
-export type ZodError = zod.ZodError;
-export type FromZodErrorOptions = {
-  maxIssuesInMessage?: number;
+function conditionallyPrefixMessage(
+  reason: string,
+  prefix: string | null,
+  prefixSeparator: string
+): string {
+  if (prefix !== null) {
+    if (reason.length > 0) {
+      return [prefix, reason].join(prefixSeparator);
+    }
+
+    return prefix;
+  }
+
+  if (reason.length > 0) {
+    return reason;
+  }
+
+  // if both reason and prefix are empty, return default prefix
+  // to avoid having an empty error message
+  return PREFIX;
+}
+
+export type FromZodIssueOptions = {
   issueSeparator?: string;
   unionSeparator?: string;
+  prefix?: string | null;
```

Comment:
dev: `prefix` may also be null if you want to disable it completely. I was unsure whether to use `null` of `false` here. I went with null.

Reply:
`null` makes more sense